app/ui-message/client/message.html

```diff
@@ -51,9 +51,9 @@
 							<i class="icon-mail" title='{{_ "Message_sent_by_email"}}' aria-label='{{_ "Message_sent_by_email"}}'></i>
 						</span>
 					{{/if}}
-					{{#if edited}}
```

Comment:
Maybe we can keep using the `edited` helper? Try to find out what is wrong with the `edited` helper instead of using `msg.editedAt`